src/components/form-editor/form-editor.component.tsx

```diff
@@ -40,6 +40,17 @@ interface ErrorProps {
   title: string;
 }
 
+interface MarkerProps {
+  startCol: number;
+  startRow: number;
+  endCol: number;
+  endRow: number;
+  className: string;
+  // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents, @typescript-eslint/no-explicit-any
+  type: string | any;
+  text: string;
+}
```

Comment:
```suggestion
import { type IMarker } from 'react-ace';

// other code...

interface MarkerProps extends IMarker {
  text: string;
}

// And then later on in the `validateJson` function:

 return {
  startRow: lineNumber,
  startCol: 0,
  endRow: lineNumber,
  endCol: 1,
  className: 'error',
  text: message,
  type: 'text' as const,
};
```

Reply:
@denniskigen this still complains about extending IMarker `Interface 'MarkerProps' incorrectly extends interface 'IMarker'.`

Reply:
Mind showing the specific error?

Reply:
Thanks, @denniskigen, this is now fixed.